cilium-health/launch/endpoint.go

```diff
@@ -118,7 +119,8 @@ func configureHealthInterface(netNS ns.NetNS, ifName string, ip4Addr, ip6Addr *n
 			name := fmt.Sprintf("net.ipv6.conf.%s.disable_ipv6", ifName)
 			// Ignore the error; if IPv6 is completely disabled
 			// then it's okay if we can't write the sysctl.
-			_ = sysctl.Write(name, "1")
+			path, _ := sysctl.ParameterPath("/proc", name)
+			_ = sysctl.WriteSysctl(afero.NewOsFs(), path, "1")
```

Comment:
The code structure makes it far from obvious (something we ought to fix), but this actually has been running inside cilium-agent for years now. So you can use the helper which uses the reconciler in the background.